examples/config/ci/config.json

```diff
@@ -0,0 +1,46 @@
+{
+  "ConnectionConfiguration": {
+    "ServerURL": "http://localhost:8065",
+    "WebSocketURL": "ws://localhost:8065",
+    "AdminEmail": "sysadmin@sample.mattermost.com",
+    "AdminPassword": "Sys@dmin-sample1"
+  },
+  "UserControllerConfiguration": {
+    "Type": "simulative",
+    "RatesDistribution": [
+      {
+        "Rate": 5.4,
+        "Percentage": 1
+      }
```

Comment:
So, did we settle on the single rate for these? Is the multi-rate used anywhere at this point?

Reply:
Should we make it our default, then? Either that or a plan to review it, I think, would be sensible.

Reply:
This number was used to match the rate of our customer's `createPost`. We should definitely review all frequencies at some point, but do we want to create a ticket for this now?